_inc/client/security/index.jsx

```diff
@@ -180,7 +178,7 @@ export const Page = ( props ) => {
 				clickableHeaderText={ true } >
 				{
 					isModuleActivated( element[0] ) || isPro ?
```

Comment:
looks like this should be a function call to `isModuleActivated` instead of a var reference